mlos_bench/mlos_bench/storage/base_storage.py

```diff
@@ -159,6 +175,11 @@ def __init__(  # pylint: disable=too-many-arguments
             )
             self._description = description
             self._opt_targets = opt_targets
+            self._ts_start = ts_start or datetime.now(UTC)
+            self._ts_end: datetime | None = None
+            self._status = Status.PENDING
```

Comment:
This should match what was stored in the backend for resumable Experiments, right?